### PR TITLE
Add section for Paypal in the Setup payment guide

### PR DESCRIPTION
**Pull request by katharinaelleke**

### CODE REVIEW

**Comment by katharinaelleke** [comment]
@davehakkens are you seeing/getting this?
Just checking how easily I can communicate with you here.
if this change could be pushed today, would be really good because I want to send out an email linking to this page 🙈

File: docs/business/Setup_Payment.md [hunk 1]
```diff
@@ -65,8 +64,58 @@ Once you are set, you can connect your Bazar account to your Stripe account.
...
+- Select “Live” mode 
+- Check the box to show the Paypal option in the Checkout
+
+![Payment methods](../assets/Business/Set%20up%20paypal%20(5).png)
+
+## Step 6: Finish!
+- Click  ‘Create’
+- you should now see your payment method listed
+
+![Payment methods](../assets/Business/Set%20up%20paypal%20(4).png)
```
Comment (davehakkens) [inline docs/business/Setup_Payment.md:115]: Would be better to not have spaces in filename. Results into weird code. Using - or _ would be preffered

**Review by davehakkens — APPROVED** [review]

**Comment by davehakkens** [comment]
Ah yeh just saw it now, i think my notifications are turned of for this

File: docs/business/Setup_Payment.md (hunk 1 above)
Comment (katharinaelleke) [reply docs/business/Setup_Payment.md:115]: yes, noted, will do in the future!
Should this be live now? I don't see any changes on the page https://community.preciousplastic.com/academy/business/Setup_Payment

**Pull request merged**